core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the preferences page, for use in Protractor
+ * tests.
+ */
+
+var PreferencesPage = function() {
+  var USER_PREFERENCES_URL = '/preferences';
+  var editorRoleEmailsCheckBox = element(by.model('canReceiveEditorRoleEmail'));
```

Comment:
We usually use by.css with prefix .protractor-test-, so could you add such a CSS class to that checkbox? That way, devs can change the variable used for Angular model binding without breaking anything.

Ditto below.

Reply:
Actually, in the `preferences.html` file these checkboxes are defined as `ng-model` element. So, as per my understanding, these should be located using by.model() and not by.css().